lib/grpc/GrpcInitService.ts

```diff
@@ -74,6 +74,28 @@ class GrpcInitService {
     }
     this.initService.pendingCall = false;
   }
+
+  /**
+   * See [[InitService.restoreNode]]
+   */
+  public restoreNode: grpc.handleUnaryCall<xudrpc.RestoreNodeRequest, xudrpc.RestoreNodeResponse> = async (call, callback) => {
+    if (!this.isReady(this.initService, callback)) {
+      return;
+    }
+    try {
+      const { restoredLndWallets, restoredRaiden } = await this.initService.restoreNode(call.request.toObject());
+      const response = new xudrpc.RestoreNodeResponse();
+      if (restoredLndWallets) {
+        response.setRestoredLndsList(restoredLndWallets);
+      }
+      response.setRestoredRaiden(restoredRaiden);
```

Comment:
Should there also be a conditional to prevent setting `restoredRaiden` when it's undefined (analogically to `restoredLndWallets`?

Reply:
Actually in double checking this code, even the first check shouldn't be necessary, as these values will never be `undefined`. I also simplified the code a bit in InitService.ts to remove some unnecessary `let` statements.